dev.pulsar_edit.Pulsar.json

```diff
@@ -0,0 +1,91 @@
+    "base-version": "22.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "pulsar.sh",
+    "rename-desktop-file": "pulsar.desktop",
+    "rename-icon": "pulsar",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.secrets",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=ELECTRON_TRASH=gio",
+        "--env=TMPDIR=/var/tmp"
+    ],
+    "modules": [
+        {
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": ["x86_64"],
+                    "url": "https://github.com/pulsar-edit/pulsar/releases/download/v1.101.0-beta/Linux.pulsar_1.101.0-beta_amd64.deb",
+                    "sha256": "d73d3c4a43fa1c86690ad6b13861d0eaf7a436015e327b960e1d821453c3fa39"
+                },
+                {
+                    "type": "file",
+                    "only-arches": ["aarch64"],
+                    "url": "https://github.com/pulsar-edit/pulsar/releases/download/v1.101.0-beta/ARM.Linux.pulsar_1.101.0-beta_arm64.deb",
+                    "sha256": "4fac74aaf9109795c82a560996748542c9c427865f91aa6391f87ad05b150af7"
+                },
+                {
+                    "type": "file",
+                    "path": "dev.pulsar_edit.Pulsar.metainfo.xml"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "pip3",
+                    "commands": [
+                        "export PYTHONUSERBASE=\"$XDG_DATA_HOME/python\"",
+                        "exec /usr/bin/pip3 \"$@\""
+                    ]
+                }
+            ],
+            "modules": [
+                "shared-modules/libsecret/libsecret.json"
+            ],
+            "name": "pulsar",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x Linux.pulsar_1.101.0-beta_amd64.deb || ar x ARM.Linux.pulsar_1.101.0-beta_arm64.deb",
+                "tar -xf data.tar.xz",
+                "cp -r usr/* /app",
+                "cp -r opt/* /app",
+                "install pip3 /app/bin",
+                "desktop-file-edit --set-key=Exec --set-value='pulsar.sh %U' /app/share/applications/pulsar.desktop",
+                "echo -e '#!/bin/sh\n/app/bin/zypak-wrapper.sh /app/Pulsar/pulsar --in-process-gpu --executed-from=\"$(pwd)\" --pid=$$ $@' > /app/bin/pulsar.sh",
+                "chmod +x /app/bin/pulsar.sh",
+                "install -Dm644 dev.pulsar_edit.Pulsar.metainfo.xml /app/share/appdata/dev.pulsar_edit.Pulsar.metainfo.xml",
+                "mkdir -p /app/extensions"
+            ]
+        },
+        {
+            "name": "xprop",
```

Comment:
this should come above the module that represent the main app. This would be logical. Also you really need it?